cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,1364 @@ Cask Data Application Platform Release Notes
+
+- `CDAP-37 <https://issues.cask.co/browse/CDAP-37>`__ -
+  Inject hconf in AbstractSparkContext by guice module ConfigModule.
+
+- `CDAP-347 <https://issues.cask.co/browse/CDAP-347>`__ -
+  User can use datasets in beforeSubmit and afterFinish.
+
+- `CDAP-362 <https://issues.cask.co/browse/CDAP-362>`__ -
+  Enabled Spark job listing and job page in Distributed mode.
+
+- `CDAP-380 <https://issues.cask.co/browse/CDAP-380>`__ -
+  CDAP Flow to use configurer style API.
+
+- `CDAP-419 <https://issues.cask.co/browse/CDAP-419>`__ -
+  Added automation and restructuring of the building of documentation.
+
+- `CDAP-585 <https://issues.cask.co/browse/CDAP-585>`__ -
+  Changes to Spark program runner to use File dataset in Spark.
+  Spark programs can now use file-based datasets.
+
+- `CDAP-593 <https://issues.cask.co/browse/CDAP-593>`__ -
+  Spark no longer determines the mode through MRConfig.FRAMEWORK_NAME.
+
+- `CDAP-595 <https://issues.cask.co/browse/CDAP-595>`__ -
+  Refactored SparkRuntimeService and SparkProgramWrapper.
+
+- `CDAP-665 <https://issues.cask.co/browse/CDAP-665>`__ -
+  Documentation received a product-specifc 404 Page.
+
+- `CDAP-683 <https://issues.cask.co/browse/CDAP-683>`__ -
+  Changed all README files from markdown to rst format.
+
+- `CDAP-722 <https://issues.cask.co/browse/CDAP-722>`__ -
+  Added a way to distinguish managed and unmanaged file datasets.
+
+- `CDAP-1008 <https://issues.cask.co/browse/CDAP-1008>`__ -
+  Added ability to restart a specific CDAP master service.
+
+- `CDAP-1132 <https://issues.cask.co/browse/CDAP-1132>`__ -
+  Improved the CDAP Doc Search Result Sorting.
+
+- `CDAP-1141 <https://issues.cask.co/browse/CDAP-1141>`__ -
+  Automated running CDAP integration tests using cdap-integration-test.
+
+- `CDAP-1143 <https://issues.cask.co/browse/CDAP-1143>`__ -
+  Removed the AuthenticatedHttpHandler.
+
+- `CDAP-1175 <https://issues.cask.co/browse/CDAP-1175>`__ -
+  Resolved Spark classpath issues in distributed mode.
+
+- `CDAP-1351 <https://issues.cask.co/browse/CDAP-1351>`__ -
+  Use ANTLR for CLI command parsing and autocompletion.
+
+- `CDAP-1416 <https://issues.cask.co/browse/CDAP-1416>`__ -
+  Added links to upper level pages on Docs.
+
+- `CDAP-1572 <https://issues.cask.co/browse/CDAP-1572>`__ -
+  Standardized Id classes.
+
+- `CDAP-1583 <https://issues.cask.co/browse/CDAP-1583>`__ -
+  Refactored InMemoryWorkerRunner and ServiceProgramRunnner after ServiceWorkers were removed.
+
+- `CDAP-1615 <https://issues.cask.co/browse/CDAP-1615>`__ -
+  Create HBase 1.0 compatibility module.
+
+- `CDAP-1801 <https://issues.cask.co/browse/CDAP-1801>`__ -
+  Bumped the version of hbase in cdap-data-fabric to use hbase98.
+
+- `CDAP-1806 <https://issues.cask.co/browse/CDAP-1806>`__ -
+  Added Spark job completion status for empty dummy programs.
+
+- `CDAP-1853 <https://issues.cask.co/browse/CDAP-1853>`__ -
+  Metrics processor scaling receives a master services restart.
```

Comment:
do we have to keep the same description as jira ? otherwise ``Metrics processor scaling no longer needs a master services restart`` is better @abaranau  ?